### DISCUSSION TITLE
Duplicating Apps

### DISCUSSION THREAD

**Discussion by sohailna** [Budibase/budibase#2921] [Help] [answered]
I've created a template dashboard that I need to be able to duplicate as more clients are on-boarded, however I couldn't find the functionality in the portal area.

Is exporting and then importing the template dashboard the only way to do this?

If so, will the imported app be identical in every way inc. database structure & data, or will some aspects be missing? 

edit: currently using Bb Cloud.

**Comment by mjashanks** [comment, marked as answer]
Yes... export then import is the best way.

All the data will be included in the export.
